packages/client/test/net/peer/peer.spec.ts

```diff
@@ -1,10 +1,7 @@
-import { EventEmitter } from 'events'
-import * as td from 'testdouble'
 import { assert, describe, it } from 'vitest'
 
 import { Config } from '../../../src/config'
 import { Peer } from '../../../src/net/peer'
-import { Event } from '../../../src/types'
 
 describe('[Peer]', () => {
   const config = new Config({ accountCache: 10000, storageCache: 1000 })
```

Comment:
It still feels very weird to me that we now have tests for an abstract class that Typescript doesn't think you can invoke.  In the compiled Javascript, this `Peer` class is just a normal class extending `EventEmitter` but we now have these ugly red squiggles to deal with.  Why is Typescript even letting this pass?